# motor_commands = mixer_matrix * dof_commands
mixer_matrix: [
  +0.0, +0.0, +1.0, +1.0, +0.0, +1.0, +0.0, +0.0,
  +0.0, +0.0, -1.0, +1.0, +0.0, -1.0, 0.0, 0.0,
  +0.0, +0.0, +1.0, +1.0, +0.0, -1.0, +0.0, +0.0,
  +0.0, +0.0, -1.0, +1.0, +0.0, +1.0, +0.0, +0.0,
  -1.0, -1.0, +0.0, +0.0, +1.0, +0.0, +0.0, +0.0,
  -1.0, +1.0, +0.0, +0.0, -1.0, +0.0, +0.0, +0.0,
  +1.0, -1.0, +0.0, +0.0, -1.0, +0.0, +0.0, +0.0,
  +1.0, +1.0, +0.0, +0.0, +1.0, +0.0, +0.0, +0.0,
  ]

limit_matrix: [
  +0.0, +0.0, +1.0, +1.0, +0.0, +1.0, +0.0, +0.0,
  +0.0, +0.0, +1.0, +1.0, +0.0, +1.0, 0.0, 0.0,
  +0.0, +0.0, +1.0, +1.0, +0.0, +1.0, +0.0, +0.0,
  +0.0, +0.0, +1.0, +1.0, +0.0, +1.0, +0.0, +0.0,
  +1.0, +1.0, +0.0, +0.0, +1.0, +0.0, +0.0, +0.0,
  +1.0, +1.0, +0.0, +0.0, +1.0, +0.0, +0.0, +0.0,
  +1.0, +1.0, +0.0, +0.0, +1.0, +0.0, +0.0, +0.0,
  +1.0, +1.0, +0.0, +0.0, +1.0, +0.0, +0.0, +0.0,
]
